framework/src/controller/controller.js

```diff
@@ -227,15 +227,25 @@ class Controller {
 			JSON.stringify({ config: this.config, moduleOptions: options }),
 		];
 
-		const child = child_process.fork(program, parameters);
+		const forkedProcessOptions = {};
+		const maxPort = 20000;
+		const minPort = 10000;
+		process.env.NODE_DEBUG
```

Comment:
Can you add a comment explaining why this is required, pls?

Reply:
Added. Resolving.